starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,51 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css" />
+  <script src="main.js"></script>
 </head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <header><div class = "presentation1"><div class="image1"> <img src = "../apple-pie-starter-code/images/apple-pie.jpg" alt="Apple pie" style="width:100%;"></div>
```

Comment:
for this one there is a trick, you can create a div, with the h1 inside, and add the following styling code : 

```css

#header_soup {
  background-image: url("../french-onion-soup-starter-code/images/soup.jpg");
  background-size: cover;
  height: 600px;
  margin: 0 5% 0 5%;
  background-attachment: fixed;
}
``` 

as you can see I used the background-image property (so it was way easier to add the text and to play with it) and to resize the image without struggling with cropping or stuff like this, just use the "background-size : cover" :)